hr_holidays_from_ics/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright © 2019 Andrei Levin <andrei.levin@didotech.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': "HR Holidays from ics",
+    'version': '12.0.0.0.0',
+    'license': 'AGPL-3',
+    'category': 'Human Resources',
+    'author': "Andrei Levin, "
+              "Odoo Community Association (OCA)",
+    'summary': """
+        Load holidays from URL that points to iCal file (*.ics)""",
+    'website': "http://www.didotech.com",
```

Comment:
https://github.com/OCA/hr

Reply:
I don't understand this

Reply:
website URL should be `https://github.com/OCA/hr`

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Reply:
Ok